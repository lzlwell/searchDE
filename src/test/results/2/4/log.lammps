LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26836 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -903.62965   -903.62965      17.1317           10           50 
       1   -1055.8736   -1055.8736      17.1317           10           50 
       2    -1114.004    -1114.004      17.1317           10           50 
       3   -1136.9177   -1136.9177      17.1317           10           50 
       4   -1149.9579   -1149.9579      17.1317           10           50 
       5   -1158.2744   -1158.2744      17.1317           10           50 
       6   -1163.2575   -1163.2575      17.1317           10           50 
       7   -1169.1024   -1169.1024      17.1317           10           50 
       8    -1172.757    -1172.757      17.1317           10           50 
       9   -1177.4503   -1177.4503      17.1317           10           50 
      10   -1179.6145   -1179.6145      17.1317           10           50 
Loop time of 0.377865 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -903.629652697     -1177.45029374     -1179.61450276
  Force two-norm initial, final = 1939.23 21.5783
  Force max component initial, final = 1335.05 9.63638
  Final line search alpha, max atom move = 0.00422904 0.0407526
  Iterations, force evaluations = 10 11

Pair  time (%) = 0.153417 (40.6009)
Neigh time (%) = 0.220896 (58.4589)
Comm  time (%) = 0.00150228 (0.39757)
Outpt time (%) = 0.000133991 (0.0354601)
Other time (%) = 0.00191617 (0.507105)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2398 ave 2398 max 2398 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46986 ave 46986 max 46986 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46986
Ave neighs/atom = 156.62
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
