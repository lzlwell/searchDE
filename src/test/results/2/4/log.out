LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26836 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -903.62965   -903.62965      17.1317           10           50 
       1   -1055.8736   -1055.8736      17.1317           10           50 
       2    -1114.004    -1114.004      17.1317           10           50 
       3   -1136.9177   -1136.9177      17.1317           10           50 
       4   -1149.9579   -1149.9579      17.1317           10           50 
       5   -1158.2744   -1158.2744      17.1317           10           50 
       6   -1163.2575   -1163.2575      17.1317           10           50 
       7   -1169.1024   -1169.1024      17.1317           10           50 
       8    -1172.757    -1172.757      17.1317           10           50 
       9   -1177.4503   -1177.4503      17.1317           10           50 
      10   -1179.6145   -1179.6145      17.1317           10           50 
Loop time of 0.377865 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -903.629652697     -1177.45029374     -1179.61450276
  Force two-norm initial, final = 1939.23 21.5783
  Force max component initial, final = 1335.05 9.63638
  Final line search alpha, max atom move = 0.00422904 0.0407526
  Iterations, force evaluations = 10 11

Pair  time (%) = 0.153417 (40.6009)
Neigh time (%) = 0.220896 (58.4589)
Comm  time (%) = 0.00150228 (0.39757)
Outpt time (%) = 0.000133991 (0.0354601)
Other time (%) = 0.00191617 (0.507105)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2398 ave 2398 max 2398 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  46986 ave 46986 max 46986 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 46986
Ave neighs/atom = 156.62
Neighbor list builds = 10
Dangerous builds = 10
